src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -591,6 +600,31 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         return api.getUsersAPI(false).getAvatarUrlWithAccessToken(userId, file.id);
     };
 
+    handleAnnotationSelect = (annotationId: string, { fileVersionId, location }: AnnotationSelectMeta): void => {
+        const { emitAnnotatorActiveChangeEvent, file, getAnnotationsMatchPath, history, onVersionChange } = this.props;
+        const { feedItems } = this.state;
+        const currentFileVersionId = getProp(file, 'file_version.id');
+
+        const match = getAnnotationsMatchPath(history);
+        const { params: { fileVersionId: currentPreviewFileVersionId = currentFileVersionId } = {} } = match || {};
+
+        emitAnnotatorActiveChangeEvent(annotationId);
+
+        if (fileVersionId !== currentPreviewFileVersionId) {
```

Comment:
What happens if we go from oldVersion -> oldVersion instead of currentVersion -> oldVersion?

Reply:
In this case, we rely on `emitAnnotatorActiveChangeEvent` to do it's job and update the AnnotatorContext with the new `activeAnnotationId` and we bypass the `onVersionChange` logic

Reply:
That seems to be referring to annotations within the same version. The scenario I'm proposing is as follows: The user is viewing version 2 of 3, then selects an annotation on version 1 (the oldest version).